src/main/kotlin/skytils/skytilsmod/features/impl/misc/ContainerSellValue.kt

```diff
@@ -135,21 +163,86 @@ object ContainerSellValue {
             return this.displayName
         }
 
+    private val textLines = mutableListOf<String>()
+    private var totalContainerValue: Double = 0.0
+    private var lines: Int = 0
+
+    private var ticks = 0
+
+    private fun shouldRenderGuiComponent() : Boolean {
+        val gui = mc.currentScreen
+        val container = if (mc.currentScreen is GuiChest) mc.thePlayer.openContainer as ContainerChest else return false
+        val chestName = container.lowerChestInventory.name
+
+        // Ensure that the gui element should be shown for the player's open container
+        return Skytils.config.containerSellValue && gui is GuiChest && isChestNameValid(chestName)
+    }
+
+    /**
+     * Compatibility with NEU's storage overlay
+     */
+    @SubscribeEvent(priority = EventPriority.LOWEST, receiveCanceled = true)
+    fun onGuiScreenDrawPre(event: GuiScreenEvent.DrawScreenEvent.Pre) {
+        if(event.isCanceled && NEUCompatibility.isStorageMenuActive && shouldRenderGuiComponent()) {
+            // NEU cancels this event when it renders the storage overlay,
+            // which means that the BackgroundDrawnEvent isn't called.
+            renderGuiComponent()
+        }
+    }
+
     /**
-     * Renders the sell value overlay when in a valid GUI.
+     * Standard rendering of the gui component when NEU's storage overlay is not active.
      *
      * Note: It is important that Minecraft Forge's `BackgroundDrawnEvent` is used instead of Skytils's
      * GuiContainerEvent.BackgroundDrawnEvent because Skytils's event is called before the overlay rectangle is drawn,
      * causing the text to be dimmed.
      */
     @SubscribeEvent
-    fun onBackgroundDrawn(event: GuiScreenEvent.BackgroundDrawnEvent) {
+    fun onPostBackgroundDrawn(event: GuiScreenEvent.BackgroundDrawnEvent) {
+        if(!NEUCompatibility.isStorageMenuActive && shouldRenderGuiComponent()) renderGuiComponent()
+    }
+
+    /**
+     * Renders the sell value overlay.
+     * @see onPostBackgroundDrawn
+     * @see onGuiScreenDrawPre
+     */
+    fun renderGuiComponent() {
+        // Translate and scale manually because we're not rendering inside the GuiElement#render() method
+        val stack = UMatrixStack()
+        stack.push()
+        stack.translate(element.actualX, element.actualY, 0f)
+        stack.scale(element.scale, element.scale, 0f)
```

Comment:
i think this will result in the same bug as the one for ch map? not sure

Reply:
I haven't experienced any issues with it and it conforms to the size of the box in the location edit gui, so it should be fine.